### PR TITLE
Update Javascript Information, little cleaner formatting IMO.

### PR DESCRIPTION
**Pull request by brandonlehmann**

### CODE REVIEW

File: api/daemon-json-rpc-api.md [hunk 1]
```diff
@@ -68,7 +72,7 @@ Parameter            | Description
 `<service port>`     | Daemon RPC port, by default it is bound to 11898 port, but it can be manually bound to any port you want
 
 
-## getblockcount
+### getblockcount
```
Comment (ar-x) [inline api/daemon-json-rpc-api.md]: This is a convention used by the static site generator. Things in `##` get auto added to the left sidebar, while things in `###` remain in page. We want to use `##` here. If this format looks better in GitHub I'll update the static site generator in a bit, thoughts?

Comment (ar-x) [reply api/daemon-json-rpc-api.md]: You can follow [this](https://github.com/turtlecoin/turtlecoin-api-docs-site/#setup) to run the frontend side by side with the docs content.

Comment (brandonlehmann) [reply api/daemon-json-rpc-api.md]: Fair enough. I'll revert those.

Comment (SoreGums) [reply api/daemon-json-rpc-api.md]: bump ;)

**Pull request closed without merging**